README.md

```diff
@@ -14,9 +14,9 @@ piece of cake. Best of all? It makes development fun!
 
 --------------------
 
-##### AngularJS will be moving to Long Term Support (LTS) mode on July 1st 2018: [Find out more](https://docs.angularjs.org/misc/version-support-status)
+**On July 1, 2018 AngularJS entered a 3 year Long Term Support period: [Find out more]**(https://docs.angularjs.org/misc/version-support-status)
```

Comment:
The closing `**` must be either before the markdown link (i.e. before the opening `[`) or after the markdown link (i.e. after the closing `)`).